package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "mes-documents-service",
+  "version": "0.1.0",
+  "author": "DVSA",
+  "license": "MIT",
+  "description": "Microservice responsible for responsible for generating documents, such as test results for Candidates.",
+  "homepage": "https://github.com/dvsa/mes-documents-service#README.md",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dvsa/mes-documents-service"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@types/aws-lambda": "^8.10.18",
+    "@types/aws-sdk": "^2.7.0",
+    "@types/chai": "^4.1.7",
```

Comment:
`chai` and `cucumber` can probably go unless we're doing BDD tests from the outset.